openverifiablellm/utils.py

```
@@ -3,29 +3,82 @@
+def compute_merkle_root(
+    file_path: Union[str, Path],
+    chunk_size: int = MERKLE_CHUNK_SIZE_BYTES,
+) -> str:
+    """
+    Compute Merkle root of a file using chunk-level hashing.
+    """
+
+    path = Path(file_path)
+
+    if not path.exists():
+        raise FileNotFoundError(f"{path} not found")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)